src/main/java/net/tropicraft/core/registry/EntityRegistry.java

```diff
@@ -44,15 +52,20 @@ public static void init() {
 		registerEntity(EntityChair.class, "beachChair", 120, 10, true);
 		registerEntity(EntityUmbrella.class, "beachUmbrella", 120, 10, false);
 		registerEntity(EntityCoconutGrenade.class, "CoconutBomb", 120, 5, true);
+		registerEntity(EntityTropicalFish.class, "fish", 80, 3, true);
+		registerEntity(EntityManOWar.class, "mow", 64, 3, true);
+		registerEntity(EntityEagleRay.class, "eagleray", 80, 1, true);
+		registerEntity(EntitySeaUrchin.class, "seaurchin", 64, 3, true);
+		registerEntity(EntitySeaUrchinEgg.class, "SeaUrchinEgg", 64, 3, false);
+		registerEntity(EntityStarfish.class, "starfish", 64, 3, false);
+		registerEntity(EntityStarfishEgg.class, "StarfishEgg", 64, 3, false);
+		//registerEntity(EntityWallStarfish.class, "WallStarfish", 160, Integer.MAX_VALUE, false);
```

Comment:
This line can be deleted entirely, wall entities are now their own thing (see a few lines below)